ci/lint.dart

```diff
@@ -89,6 +102,25 @@ Future<List<String>> dirContents(String repoPath) {
   return completer.future;
 }
 
+Future<bool> shouldIgnoreFile(String path) async {
+  if (path.contains('/third_party/')) {
+    return true;
+  } else {
+    final RegExp exp = RegExp(r'//.*FLUTTER_NOLINT');
+    await for (String line in File(path.substring(6))
+        .openRead()
+        .transform(utf8.decoder)
+        .transform(const LineSplitter())) {
+      if (exp.hasMatch(line)) {
+        return true;
+      } else if (line.length > 0 && line[0] != '\n' && line[0] != '/') {
```

Comment:
Could you add a comment here about what this case is doing?

Reply:
Done.